main.sh

```diff
@@ -16,4 +16,106 @@
 #
 # Copyright 2023 The Kepler Contributors
 #
-./common.sh $1
+
+set -eu -o pipefail
+
+PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+declare -r PROJECT_ROOT
+declare -r BIN_DIR="$PROJECT_ROOT/tmp/bin"
+
+source "$PROJECT_ROOT/.env"
+source "$PROJECT_ROOT/lib/utils.sh"
+
+declare -r CTR_CMD=${CTR_CMD:-docker}
+declare -r CLUSTER_PROVIDER=${CLUSTER_PROVIDER:-kind}
+declare -r REGISTRY_PORT=${REGISTRY_PORT:-5001}
+declare -r CONFIG_OUT_DIR=${CONFIG_OUT_DIR:-"_output/generated-manifest"}
+declare -r PROMETHEUS_ENABLE=${PROMETHEUS_ENABLE:-true}
+
+declare -r ESTIMATOR_REPO=${ESTIMATOR_REPO:-quay.io/sustainable_computing_io}
```

Comment:
I think we can remove `ESTIMATOR_REPO` and `MODEL_SERVER_REPO` as it's not used anywhere. @SamYuan1990 Correct me if I am wrong

Reply:
yep, I suppose we can remove it. + @sunya-ch here as ESTIMATOR related maintainer, please help double confirm we can remove this env declare.

Reply:
The operator uses offline trained models so we can remove the estimator and the model server.

Reply:
for now we can skip estimator